packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/long-form-vpc.yaml

```diff
@@ -0,0 +1,48 @@
+Mappings:
+  RegionMap:
+    region-1:
+      HVM64: name1
+      HVMG2: name2
+Conditions:
+  AlwaysTrueCond:
+    Fn::Not:
+    - Fn::Equals:
+      - Ref: AWS::Region
+      - completely-made-up-region
```

Comment:
How about one condition that uses `Fn::Not`, `Fn::Equals`, `Fn::And` and `Fn::Or` in its expression? I don't see much gain in this test from having 2 of these. The `Condition` syntax we already have covered with JSON tests.